doc/manuals/user/oneshot_subscription.md

```diff
@@ -1,60 +0,0 @@
-# Oneshot Subscription
```

Comment:
Remove also from mkdocs.yml file and users/README.md

Reply:
Done in 89be00c229c5cf8afb8245aafc9a0ab077f9a369